config/brakeman.ignore

```diff
@@ -18,6 +18,25 @@
       "user_input": "params[:path]",
       "confidence": "Weak",
       "note": ""
+    },
+    {
+      "warning_type": "Redirect",
+      "warning_code": 18,
+      "fingerprint": "e1e5e4ef3cefe762741c95668c9451d928b2c179773a866849a4897d34752af1",
+      "message": "Possible unprotected redirect",
+      "file": "app/controllers/v0/sessions_controller.rb",
+      "line": 78,
+      "link": "http://brakemanscanner.org/docs/warning_types/redirect/",
+      "code": "redirect_to((RelayState.new(:relay_enum => params[:success_relay], :url => params[:RelayState]).logout_url + \"?success=true\"))",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "V0::SessionsController",
+        "method": "saml_logout_callback"
+      },
+      "user_input": "(RelayState.new(:relay_enum => params[:success_relay], :url => params[:RelayState]).logout_url + \"?success=true\")",
+      "confidence": "High",
+      "note": "Manually reviewed code path, and relay URL is whitelisted. Bill Ryan"
```

Comment:
We didn't need anything like this before dor login did we? Why do we need it now?

Reply:
Because we used to `redirect_to` a static setting. This
```ruby
      redirect_to Settings.saml.logout_relay + '?success=true'
```
became:
```ruby
      redirect_to @relay_state.logout_url + '?success=true'
```

I wondered why `brakeman` didn't also complain about the new way we do login when, but realized it's done from a method call which can return either a static setting or come from `RelayState`, e.g.:

```ruby
  redirect_to saml_login_relay_url + '?token=' + @session.token
  ...
    def saml_login_relay_url
      return @relay_state.login_url if current_user.nil?
      # TODO: this validation should happen when we create the user, not here
      if current_user.loa.key?(:highest) == false || current_user.loa[:highest].nil?
        log_message_to_sentry('ID.me did not provide LOA.highest!', :error)
        return @relay_state.login_url
      end

      if current_user.loa[:current] < current_user.loa[:highest]
        SAML::SettingsService.idme_loa3_url(current_user, @relay_state)
      else
        @relay_state.login_url
      end
    end
```